modules/odo-core-concepts.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-core-concepts_{context}"]
+
+= odo core concepts
+
+`{odo-title}` abstracts Kubernetes concepts into a developer friendly terminology:
+
+Application::
+An _application_ in `odo` is a classic application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that is used to perform a particular task.
++
+Examples of applications: Online Video Streaming, Hotel Reservation System, Online Shopping.
```

Comment:
```suggestion
For example, video streaming, hotel reservation, and e-commerce websites.
```
Or tack on to the end of the preceding sentence with ", such as..."

Reply:
In structured authoring, a definition or glossary entry is typically made up of a term, a standalone definition and then other material including examples - see https://tdg.docbook.org/tdg/5.0/glossentry.html

So I would prefer to keep the examples separate from the definition - I will change the case for the examples

Reply:
Examples of components include an API back-end, a web interface and a payment back-end.

Examples of services include PostgreSQL, MySQL, Redis and RabbitMQ.